src/specialOffer/SpecialOffer.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+
+import { Button, ButtonStatus } from '../button'
+import { Disclaimer } from '../disclaimer'
+import { Paragraph } from '../paragraph'
+import { SubHeader } from '../subHeader'
+import {
+  StyledSpecialOffer,
+  StyledSpecialOfferButtonLine,
+  StyledSpecialOfferButtons,
+  StyledSpecialOfferContent,
+  StyledSpecialOfferFigure,
+  StyledSpecialOfferText,
+} from './SpecialOffer.style'
+
+type Picture = Readonly<{
+  url: string
+  alt: string
+}>
+
+type Link = Readonly<{
+  label: string
+  url: string
+}>
+
+type Button = Readonly<{
+  label: string
+  onClick: () => void
+}>
+
+type CTALinks = Array<Link>
+type CTAButtons = Array<Button>
+
+export type SpecialOfferProps = Readonly<{
+  picture: Picture
+  title: string
+  description: string
+  informationLink?: Link
+  actions?: CTALinks | CTAButtons
```

Comment:
I'm wondering: there is no way to mix up links + buttons in the actions?
I don't know if we want to add this flexibility but you could remove your `CTALinks` & `CTAButtons` types and do this:
```suggestion
  actions?: Array<Link | Button>
```
But maybe you force link-only or button-only actions on purpose, if so you can ignore my comment 👍

Reply:
Yes, as for now we want to be sure we don't mix the two, I will keep the initial typing, thanks for your suggestion!